src/nft/components/mint/NFTCollectionTitle.tsx

```diff
@@ -12,5 +12,15 @@ export function NFTCollectionTitle({ className }: NFTCollectionTitleReact) {
     return null;
   }
 
-  return <div className={cn('pt-4 pb-1', text.title1, className)}>{name}</div>;
+  return (
+    <div
+      className={cn(
+        'overflow-hidden text-ellipsis pt-4 pb-1',
+        text.title1,
+        className,
```

Comment:
Nit: I feel like the order of classNames should always be:

`themes -> hard coded tailwind classes -> className`

Reply:
I may have a few places like this, I can revisit this in a separate PR